model/models.py

```diff
@@ -128,3 +130,87 @@ def forward(self, sub, rel):
 
 		score = torch.sigmoid(x)
 		return score
+
+	# Documentation
+
+
+class CompGCN_CTKGC(CompGCNBase):
+		'''
+		Implements the CTKGC Scoring Function as in the paper https://link.springer.com/article/10.1007/s10489-021-02438-8 and writes to the console the total Parameter uesd in the Network
+
+		The scoring Function works by
+			1. Build the Entity-relation matrix by multiplying sub_emb.transpose(1.0) and rel_emb
+			2. Then we perform a Convolution on Entity-relation matrix
+			3. We Aggregate all the obtained feature,
+			4. Convert the above feature maps to a 1D Vector
+			5. We project our 1D vector onto a candidate objects to obtain the predicted object embedding
+				1. We do 5. for all our entities (candidate objects)
+			5. Use the sigmoid function to calculate the score of the object embedding
```

Comment:
I guess this should be a 6
```suggestion
			6. Use the sigmoid function to calculate the score of the object embedding
```